### DISCUSSION TITLE
Dịch constructor

### DISCUSSION THREAD

**Discussion by hnguyentt** [mlbvn/handson-ml2-vn#129] [Terminology] [answered]
Context:
> The constructor accepts `**kwargs` and passes them to the parent constructor, which handles standard hyperparameters: the `name` of the loss and the `reduction` algorithm to use to aggregate the individual instance losses.

Mình đề xuất dịch:
* constructor: hàm tạo
* parent constructor: hàm tạo lớp cha

**Comment by ngcthuong** [comment]
Từ này mình nghĩ bên lập trình họ chắc có dịch rồi chứ nhỉ? Không biết có ai có tham khảo không?

**Reply by hnguyentt** [reply]
Em thấy trong OOP họ dịch là hàm tạo hoặc hàm dựng: https://vi.wikipedia.org/wiki/H%C3%A0m_t%E1%BA%A1o_(l%E1%BA%ADp_tr%C3%ACnh_h%C6%B0%E1%BB%9Bng_%C4%91%E1%BB%91i_t%C6%B0%E1%BB%A3ng)

**Reply by minhduc0711** [reply]
Em hay dịch từ này là `hàm khởi tạo`, nhưng em thấy `hàm tạo` cũng ổn.

**Reply by rootonchair** [reply]
Trong giáo trình và dùng phổ biến em thấy dịch là `hàm khởi tạo`

**Comment by minhduc0711** [comment, marked as answer]
Vậy chốt nhé:
* constructor: hàm khởi tạo
* parent constructor: hàm khởi tạo lớp cha
